functions/package.json

```diff
@@ -14,11 +14,12 @@
   },
   "dependencies": {
     "@googlemaps/google-maps-services-js": "^2.3.1",
+    "csv-stringify": "",
     "firebase-admin": "^8.6.0",
     "firebase-functions": "^3.3.0",
+    "flag-icon-css": "^3.4.6",
```

Comment:
Not sure why changes to package and package-lock are necessary, since you're linking directly to the CDN-hosted css.